gui-js/apps/minsky-electron/src/app/managers/CommandsManager.ts

```
@@ -1308,6 +1370,45 @@ export class CommandsManager {
+    let state;
+    switch (process.platform) {
+    case 'win32':
+      state={system: 'windows', distro: '', version: '', arch:'', previous: ''};
+      break;
+    case 'darwin':
+      state={system: 'macos', distro: '', version: '', arch: `${process.arch}`, previous: ''};
+      break;
+    case 'linux':
+      state={system: 'linux', distro: '', version: '',arch:'', previous: ''};
+      // figure out distro and version from /etc/os-release
+      let aexec=promisify(exec);
```

Biome (2.4.9)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)